v2x_ros_driver/launch/v2x_ros_driver.launch.py

```diff
@@ -76,32 +84,41 @@ def generate_launch_description():
         on_exit= Shutdown()
     )
 
-    ros2_cmd = launch.substitutions.FindExecutable(name='ros2')
+    ros2_cmd = launch.substitutions.FindExecutable(name="ros2")
 
-    process_configure = launch.actions.ExecuteProcess(
-        cmd=[ros2_cmd, "lifecycle", "set", "/v2x_ros_driver", "configure"],
+    process_configure_v2x_ros_driver_node = launch.actions.ExecuteProcess(
+        condition=IfCondition(
```

Comment:
I think we can change this to `condition=IfCondition(LaunchConfiguration('enable_v2x_driver_lifecycle'))`. Though we might need to change the default value of `enable_v2x_driver_lifecycle` from `false` to `False`